controllers/routemap.go

```diff
@@ -216,6 +221,36 @@ versionLoop:
 	return versionsAvailable
 }
 
+// computeSignature computes and sets the signature for a particular version
+// signature is based on the spec section
+func computeSignature(v version) (uint64, error) {
+	// get all resources of a version
+	resources := []interface{}{}
+	for _, resource := range v.Resources {
+		obj, err := appInformers[resource.GVRShort].Lister().ByNamespace(*resource.Namespace).Get(resource.Name)
+		if err != nil {
+			return 0, fmt.Errorf("cannot get resource: %e", err)
+		}
+
+		// TODO: NestedFieldCopy() or NestedFieldNoCopy()?
```

Comment:
NestedFieldNoCopy should be fine; we have a copy of the object no reason to copy again